### ISSUE TITLE
The 'sklearn' PyPI package is deprecated, use 'scikit-learn'  rather than 'sklearn' for pip commands.

### ISSUE THREAD

**Issue by kmoonn** [facebookresearch/CrypTen#512]
please use pip install scikit-learn rather than pip install sklearn.
Can you change the package name of sklearn for the dependency library? 
Because it prevents me from downloading the crypten library.

**Comment by preejackie** [comment]
Yeah. Please sklearn is deprecated. I can't able to install crypten library, with pip install crypten.

**Comment by aaallami** [comment]
Same is here, I cannot install it

**Comment by trajore** [comment]
you can install it by setting this env variable to true
```
export SKLEARN_ALLOW_DEPRECATED_SKLEARN_PACKAGE_INSTALL=True
```

**Comment by d41sys** [comment]
Huh, it seems the author does not attend this project anymore.
